engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1958,12 +1957,13 @@ class IrToTruffle(
     }
 
     private def buildForeignBody(
-      language: ForeignLanguage,
+      language: String,
       code: String,
       argumentNames: List[String],
       argumentSlotIdxs: List[Int]
     ): RuntimeExpression = {
-      val src       = language.buildSource(code, scopeName)
+      val src =
+        Source.newBuilder("epb", language + "#" + code, scopeName).build()
```

Comment:
Use something like `EpbLanguage.ID` instead of hard-coding "epb"?

Reply:
Too much overhead as explained to Hubert: https://github.com/enso-org/enso/pull/7882#discussion_r1426620505